easybuild/easyconfigs/p/pypy/pypy-7.3.5-x86-64.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Tarball'
+
+name = 'pypy3.7'
+version = '7.3.5'
```

Comment:
```suggestion
name = 'pypy'
version = '3.7-v7.3.5'
```

Reply:
How odd to have the source tarball called pypy3.7-v7.3.5.... wouldn't that source-tarball be the same one used to build a python2.7 based version of this?